src/website/views/components/forms/checkbox/_accessibility.pug

```diff
@@ -1,5 +1,87 @@
+
+.chi-grid
+  .chi-col.-w-sm--12.-w-md--6.-w-lg--6.-mb--3
+    .chi-card.-widget
+      .chi-card__header 
+        .chi-card__title.-text--normal Guidance for developers
+      .chi-card__content 
+        .chi-card__caption
+          ul.-text
+            li Use clear and descriptive labels for checkboxes to help users understand their purpose.
+            li Ensure each checkbox has an associated <code>label</code> element using the <code>for</code> attribute to match the <code>id</code> attribute of the checkbox.
+            li Assign unique <code>id</code> attributes to each checkbox to maintain accessibility and prevent conflicts.
+            li The <code>fieldset</code> surrounds the entire grouping of checkboxes or radio buttons. The <code>legend</code> provides a description for the grouping.
+            li Use the <code>aria-describedby</code> attribute to associate error messages with checkboxes for accessibility.
+            li Ensure that error messages are tagged with unique id attributes and referenced correctly with <code>aria-describedby</code> on corresponding checkboxes. 
+  
+  .chi-col.-w-sm--12.-w-md--6.-w-lg--6.-mb--3
+    .chi-card.-widget
+      .chi-card__header 
+        .chi-card__title.-text--normal Guidance for designers
+      .chi-card__content 
+        .chi-card__caption 
+          ul.-text
+            li Ensure focus indicators are clearly visible to aid navigation for keyboard users.
+            li Keep input labels visible, even when fields are focused.
+            li Use contrasting colors for text and backgrounds to enhance readability.
+            li Design clear error states and feedback for accessibility.
+            li Use spacing, typography, and visual cues effectively.
+            li Maintain consistency in the design of checkboxes to provide a cohesive user experience.
+
+p.-text 
+  | Find live examples in the <a href="https://a11y-style-guide.com/style-guide/section-forms.html#kssref-forms-checkboxes" target="_blank">A11y style guide</a>.  
+
+h3 Roles and attributes
+p.-text 
+  | WAI-ARIA provides a grouping role that functions similarly to fieldset and legend.
+table.chi-table
+  thead
+    tr
+      th Role
+      th Element
+      th Usage
+  tbody
+    tr
+      td <code>group</code>
+      td <code>div</code>
+      td Identifies the <code>div</code> element as a group container for the checkboxes.
+
+h3 Resources
+ul.-text
+li
+  a(href="https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/examples/checkbox/") W3 checkbox example: 
+  | Information about the accordion accessible behavior.
+li
+  a(href="https://a11y-style-guide.com/ ") A11y Style Guide:  
+  | Gives core recommendations for the accessible checkboxes. 
+li  
+  a(href="https://www.w3.org/WAI/tutorials/forms/") W3C Forms Tutorial:  
+  | Provides guidance on creating accessible forms.
+li  
+  a(href="https://www.digitala11y.com/") DigitalA11y Project: 
+  | Find examples of accessible checkboxes.
```

Comment:
@claradamcho indentation is wrong here and it is causing that the accessibility page fpor checkbox is displaying wrongly.

Let's check that everything's fine in our devenv before sending any commit.

![Screen Shot 2024-03-15 at 2 41 03 PM](https://github.com/CenturyLink/Chi/assets/17043903/973f6214-692c-4867-a62c-be6f666e0f68)